user/views.py

```diff
@@ -0,0 +1,58 @@
+import json,traceback
+import bcrypt #pip install bcrypt
+import jwt #pip install pyJWT 
+
+from .models                import User
+from devx.settings          import SECRET_KEY
+from datetime import datetime
+
+from django.views           import View
+from django.http            import JsonResponse,HttpResponse
+from django.core.exceptions import ValidationError
+from .validation            import Validate_firstname
+from .decorator             import login_decorator
+
+
+
+class SignUpView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+        try:
+            #비밀                                            번호 암호화
+            password        = data['password'].encode('utf-8')
+            password_crypt  = bcrypt.hashpw(password,bcrypt.gensalt())
+            password_crypt  = password_crypt.decode('utf-8')
+            #끝 
+            req_User = User(
```

Comment:
```suggestion
            request_user = User(
```
변수명은 명확히 알아들을 수 있는 단어려 명명해주시고, 언더바와 카멜케이스를 섞어쓰는 변수 선언 방법은 현재 없습니다 :) 파이썬에서는 클래스가 아닌 변수명에 camelCase를 사용하지 않으니 snake_case로 통일해주세요

Reply:
넵 수정하였습니다